doc/onboarding.md

```diff
@@ -38,6 +38,10 @@ onboarding session.
       * Branches in the nodejs/node repository are only for release lines
     * [See "Updating Node.js from Upstream"](./onboarding-extras.md#updating-nodejs-from-upstream)
     * Make a new branch for each PR you submit.
+    * Membership: We ask that collaborators make their membership in the Node.js
```

Comment:
Maybe specify `GitHub membership` or move into a separate GitHub section? At first is a little bit unclear.

Reply:
Just to clarify, is this a request or a requirement for collaborators?

Reply:
Maybe move to near the requirement for enabling 2FA?

Reply:
I agree, the current wording makes this sound like a requirement, which is something I would be *really* careful about.

Reply:
@cjihrig 
> Just to clarify, is this a request or a requirement for collaborators?

Having your membership public means GitHub can assign you the appropriate UI badges.
As it is now:

<img src="https://user-images.githubusercontent.com/4303/34017594-3946943c-e0db-11e7-847a-243dadf0e80c.png" width=80%>

and

<img src="https://user-images.githubusercontent.com/4303/34017604-4036ec4c-e0db-11e7-8521-5201adf2ff52.png" width=20%>

There's no indicator that you're part of the Node org. much less a member of the TSC or collaborator.

Reply:
I know. Aren't the labels that GitHub shows next to your name in these threads, along with the list in the project's README enough?

I'm fine with a request/recommendation, but would be -1 on a hard requirement.

Reply:
You're membership is already public in the sense that your name is listed [here](https://github.com/nodejs/node#collaborators) and [here](https://github.com/nodejs/node#tsc-technical-steering-committee). This just lets GitHub do their UI thing so that users can more easily identify project maintainers.

Reply:
I'm saying as it is now your label is incorrect. I'm also labeled a "contributor" but I'm not part of the org. If you were part of the org you would have the name "member".

Reply:
Ah, I thought you were a contributor :-)

Even still, I'm not sure that it would be a huge help. We have a members team with over 500 people that includes more people than the core collaborator list.

I'm mostly being argumentative on principle. It feels like Node would be overstepping its boundaries by requiring that collaborators make membership public. I'm a member of several other orgs, and none of them have such a requirement.

Reply:
> Ah, I thought you were a contributor :-)

Exactly. You proved my point.

Reply:
@cjihrig the "Owner" next to your name is only shown if the current logged-in user is in the @ nodejs org.

Reply:
The closeness in "collaborator" and "contributor" may also be part of the confusion.